### PR TITLE
Make help command show descriptions, syntax & subcommands

### PR DESCRIPTION
**Pull request by dlsf**
tl;dr /is help is now smarter

### CODE REVIEW

File: src/main/java/com/iridium/iridiumskyblock/commands/HelpCommand.java [hunk 1]
```diff
@@ -19,11 +22,17 @@
  */
 public class HelpCommand extends Command {
 
+    public String defaultCommandSyntax = "&7/is %command%";
+    public List<String> commandHelpMessage = Arrays.asList(
+        "&7Description: &b%description%",
+        "&7Syntax: &b%syntax%");
+    public String availableSubcommands = "&7Available subcommands: &b%commands%";
```
Comment (dlsf) [inline src/main/java/com/iridium/iridiumskyblock/commands/HelpCommand.java:29]: No idea if this is a good place but for some reason, I really thought it is.

**Comment by dlsf** [comment]
The subcommands can also be shown behind the syntax (e.g. `Syntax: /is bank <give/set/remove>` instead of in a separate message. Thoughts?

File: src/main/java/com/iridium/iridiumskyblock/commands/ExpelCommand.java [hunk 2]
```diff
@@ -23,7 +23,7 @@
      * The default constructor.
      */
     public ExpelCommand() {
-        super(Collections.singletonList("expel"), "Kick a visitor", "%prefix% &7/is expel [<name>]", "", true, Duration.ZERO);
+        super(Collections.singletonList("expel"), "Kick a visitor", "%prefix% &7/is expel <player>", "", true, Duration.ZERO);
```
Comment (MertUnverdi) [inline src/main/java/com/iridium/iridiumskyblock/commands/ExpelCommand.java:26]: idk about this but, i was did with [], its an optional arg so shouldn't be <> ?

File: src/main/java/com/iridium/iridiumskyblock/commands/RegenCommand.java [hunk 3]
```diff
@@ -25,7 +25,7 @@
      * The default constructor.
      */
     public RegenCommand() {
-        super(Collections.singletonList("regen"), "Regenerate your Island", "", true, Duration.ofMinutes(5));
+        super(Collections.singletonList("regen"), "Regenerate your Island", "%prefix% &7/is regen <schematic>", "", true, Duration.ofMinutes(5));
```
Comment (MertUnverdi) [inline src/main/java/com/iridium/iridiumskyblock/commands/RegenCommand.java:28]: its also optional

**Review by MertUnverdi — APPROVED** [review]

File: src/main/java/com/iridium/iridiumskyblock/commands/ExpelCommand.java (hunk 2 above)
Comment (PeachesMLG) [reply src/main/java/com/iridium/iridiumskyblock/commands/ExpelCommand.java:26]: the syntax ive been using is () is optional <> is required

Comment (dlsf) [reply src/main/java/com/iridium/iridiumskyblock/commands/ExpelCommand.java:26]: No, you used <> everywhere, for example in all commands that have a GUI but an optional argument

**Comment by PeachesMLG** [comment]
Resolve these conflicts (cuz im too lazy 2) then ill merge

**Pull request merged**